### PR TITLE
Updated FindOracle.cmake for Oracle 18 XE

### PR DESCRIPTION
**Pull request by enricodetoma**
Here is an updated CMake configuration for Oracle 18 XE.
Tested with Oracle 18 XE on Windows.

### CODE REVIEW

File: cmake/modules/FindOracle.cmake [hunk 1]
```diff
@@ -43,15 +43,16 @@ find_path(ORACLE_INCLUDE_DIR
...
+set(ORACLE_OCCI_NAMES libocci occi oraocci10 oraocci11 oraocci12 oraocci18)
+set(ORACLE_NNZ_NAMES nnz10 libnnz10 nnz11 libnnz11 nnz12 libnnz12 nnz18 libnnz18 ociw32)
 
 set(ORACLE_LIB_DIR
   ${ORACLE_HOME}
   ${ORACLE_HOME}/lib
   ${ORACLE_HOME}/sdk/lib       # Oracle SDK
   ${ORACLE_HOME}/sdk/lib/msvc
   ${ORACLE_HOME}/OCI/lib/msvc # Oracle XE on Windows
+  ${ORACLE_HOME}/OCI/lib/msvc/vc14 # Oracle XE on Windows
```
Comment (vadz) [inline cmake/modules/FindOracle.cmake]: Is this really needed here? We didn't add `vcN` subdirectories for the previous version...

Comment (enricodetoma) [reply cmake/modules/FindOracle.cmake]: Unfortunately `oraocci18.lib` is in `${ORACLE_HOME}/OCI/lib/msvc/vc14` directory, and could not be found.
I will re-check it later when I'm back at my PC, to be sure it's really needed.

Comment (enricodetoma) [reply cmake/modules/FindOracle.cmake]: You are right, `${ORACLE_HOME}/OCI/lib/msvc/vc14` is not needed.
I pushed a commit to remove it.

Comment (vadz) [reply cmake/modules/FindOracle.cmake]: Thanks for testing!

**Pull request closed without merging**